python/base_hooks/hiero_customize_export_ui.py

```diff
@@ -0,0 +1,257 @@
+# Copyright (c) 2018 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+#  UI Hook
+# ===========================
+
+import sgtk
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class HieroCustomizeExportUI(HookBaseClass):
+    """
+    This class defines methods that can be used to customize the UI of the various
```

Comment:
could we include a screenshot to illustrate as part of the docs?

Reply:
Done. Also added some overview sphinx docs for customizing UI. Basic stuff, but it's a good, brief intro with an example screenshot.